src/components/TodoItem/index.tsx

```diff
@@ -5,21 +5,26 @@ import { useDispatch } from 'react-redux';
 import { TodoItemPropsType } from './types';
 import { deleteItemSagaAction } from '../../redux/actions/todoSagaActions/deleteItemSagaAction';
 import { doneItemSagaAction } from '../../redux/actions/todoSagaActions/doneItemSagaAction';
+import PushNotification from 'react-native-push-notification';
 
 export const TodoItem: FC<TodoItemPropsType> = props => {
   const {
-    todoItem: { id, text, isDone },
+    todoItem: { id, text, isDone, notificationId },
   } = props;
 
   const dispatch = useDispatch();
 
   const textStyle = isDone ? style.doneText : style.text;
 
   const onPressDone = () => {
+    PushNotification.cancelLocalNotification(`${notificationId}`);
+
     dispatch(doneItemSagaAction({ id, isDone }));
   };
 
   const onPressDelete = () => {
+    PushNotification.cancelLocalNotification(`${notificationId}`);
```

Comment:
after alert is appeared and you press 'NO' - your item won't be deleted but notification already have been cleared

Reply:
understood, it will be fixed